posix.mak

```diff
@@ -519,7 +523,17 @@ test:
 # Changelog generation
 ################################################################################
 
-pending_changelog:
-	@echo "This command will be available soon."
+changelog/${NEXT_VERSION}.dd: ${STABLE_DMD} ../tools ../installer
+	$(STABLE_RDMD) $(TOOLS_DIR)/changed.d "v${LATEST}..upstream/stable" -o changelog/${NEXT_VERSION}.dd \
```

Comment:
Always using stable seems incorrect when the purpose of this target is trying to build the changelog for dmd/druntime/phobos PRs.